src/hrc721.ts

```diff
@@ -2,7 +2,7 @@ import { Contract } from '@harmony-js/contract'
 import { Harmony } from '@harmony-js/core'
 import { BigNumber, logger } from 'ethers'
 import { Logger } from 'ethers/lib/utils'
-import { BaseHRC721 } from './base-hrc721'
+import { BaseHRC721 } from './base-HRC721'
```

Comment:
let's keep the naming in lowercase so that we are consistent (you can do `git mv base-HRC721 base-hrc721`)